recipes/mastercard/runner.sh

```diff
@@ -8,13 +8,11 @@ VERSION=$DATE
 
 AWS_DEFAULT_REGION=us-east-1
 
-
-
 (      
     
     cd $BASEDIR
+    rm -rf input
```

Comment:
are you removing it before it's being created?

Reply:
I made changes so that it removes it after it is created and before it is created. Removing before it is created is in case it failed to clean up when it crashed.